spec/models/collection_spec.rb

```diff
@@ -25,12 +25,13 @@ def to_solr(solr_doc={}, opts={})
   let(:user) { @user }
 
   describe "#to_solr" do
-    let(:collection) { Collection.new(title: "A good title") }
+    let(:collection) { Collection.new(title: "A good title", depositor: user.user_key) }
 
     subject { collection.to_solr }
 
     it "should have title" do
       expect(subject['title_tesim']).to eq ['A good title']
+      expect(subject['depositor_tesim']).to eq [user.user_key]
```

Comment:
I would not have expected ssim too.  I wonder why one index give you two fields...